app/uk/gov/hmrc/gform/views/hardcoded/InsufficientCredentialsPage.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.views.hardcoded
+
+import play.api.data.Form
+import play.api.i18n.Messages
+import play.twirl.api.Html
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplate
+import uk.gov.hmrc.govukfrontend.views.html.components.{ ErrorLink, ErrorMessage, ErrorSummary, Fieldset, GovukErrorMessage, GovukFieldset, GovukHint, GovukLabel, GovukRadios, Legend, RadioItem, Radios, Text }
+
+class InsufficientCredentialsPage(val formTemplate: FormTemplate, form: Form[String])(implicit messages: Messages)
```

Comment:
This looks like copy of `EnrolmentAlreadyLinkedPage`. Can we rename `EnrolmentAlreadyLinkedPage` to something more general and reuse it?